From 238e3ed1e69c31337eebc2c49058c3786acc1a05 Mon Sep 17 00:00:00 2001
From: czietz <czietz@gmx.net>
Date: Sun, 10 Feb 2019 15:43:45 +0000
Subject: [PATCH 2/4] Make variable type and format string match in fmt_size().

Since a file length cannot be negative, change the size parameter to ULONG to match the "%lu" format specifiers.
---
 desk/deskinf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desk/deskinf.c b/desk/deskinf.c
index 20a7ff15..e09003ee 100644
--- a/desk/deskinf.c
+++ b/desk/deskinf.c
@@ -185,7 +185,7 @@ static BYTE *fmt_date(UWORD date, BOOL fourdigit, BYTE *pdate)
  * Note: files larger than 9999999 bytes will be displayed
  * in kbytes on narrow screens
  */
-static BYTE *fmt_size(LONG size, BOOL wide, BYTE *psize)
+static BYTE *fmt_size(ULONG size, BOOL wide, BYTE *psize)
 {
     BYTE *fmt_string;
 
-- 
2.17.0.windows.1

